db/migrations/20190605141451_create_locations.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+exports.up = (Knex) => {
+  return Knex.schema.createTable('locations', (table) => {
+    table.increments('id').primary();
+    table.text('city').notNullable();
+    table.text('state').notNullable();
+    table.index(['city', 'state'], 'idx_city_state');
```

Comment:
This index was added for performance reasons I presume?

Reply:
Correct!